### PR TITLE
add multiply function index.js

### PR DESCRIPTION
**Pull request by meetbhayani123**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -10,3 +10,7 @@ const sum = (a,b) => {
 };
 
 sum(5,6)
+
+const multiply = (a,b) => {
```
Comment (meetprolanceit) [inline index.js:14]: you have to call it also after crete or you forgot this ?

**Review by meetprolanceit — COMMENTED** [review]
do changees based on the commewntss

File: index.js (hunk 1 above)
Comment (meetbhayani123) [reply index.js:14]: okay i will and tthen i push again

**Review by meetprolanceit — APPROVED** [review]
your PR is completed

**Pull request merged**